### PR TITLE
WIP - Add French translation

### PR DESCRIPTION
**Pull request by corradio**
Refers to https://github.com/corradio/electricitymap/issues/311
TODO:
- [x] Facebook support
- [x] Make sure proper renames on master were applied here
- [x] Country name support (https://www.npmjs.com/package/i18n-iso-countries)
- [ ] ~Share translations between frontend and backend~ (later)
- [x] Review translations

### CODE REVIEW

**Comment by corradio** [comment]
@ThierryOllivero @brunolajoie can you review / improve the translations?
Note: there are 3 json files to review (one for browser content, and two for server-generated content). I will merge them into one file per language asap.

**Comment by ThierryOllivero** [comment]
What are the 3 files? lang.json , en.js and fr.js ?

**Comment by corradio** [comment]
yes!

**Comment by corradio** [comment]
Simpler for the language review:

image
image
image
image
image
image
image
image

**Comment by corradio** [comment]
Actually there was a couple of bugs that I just fixed. Here are the new tooltips:

image
image
image
image

cc @brunolajoie @ThierryOllivero

**Comment by brunolajoie** [comment]
Si t'as plus ce petit bug, ca me vas
image

**Comment by corradio** [comment]
Good catch @brunolajoie !

Please validate the new version:
image

Also: note that the facebook/twitter buttons have changed language :)

**Comment by ThierryOllivero** [comment]
I would change "combien de CO2 a été émis" by "quelle quantité de CO2 a été émise". 

I would change "voir (data sources)" by "(voir data sources)", as it is the case in English.

**Comment by corradio** [comment]
@ThierryOllivero 

image

**Comment by ThierryOllivero** [comment]
Much better! Should be ok now.

**Comment by corradio** [comment]
Européenne prends deux "n"!

J'ai traduit les pays aussi:

image

**Comment by corradio** [comment]
@nikkozzblu can you do the final review?

File: web/views/pages/index.ejs [hunk 1]
```diff
@@ -5,10 +5,14 @@
         <meta name="description" content="Electricity Map is a live visualization of where your electricity comes from and how much CO2 was emitted to produce it." />
         <meta property="og:description" content="Electricity Map is a live visualization of where your electricity comes from and how much CO2 was emitted to produce it." />
         <meta property="og:image" content="https://cloud.githubusercontent.com/assets/1655848/20340757/5ada5cf6-abe3-11e6-97c4-e68929b8a135.png" />
-        <meta property="og:title" content="Electricity Map | Live CO2 emissions of the European electricity consumption" />
+        <meta property="og:title" content="<%= __('misc.maintitle') %>" />
         <meta property="og:type" content="website" />
```
Comment (nikkozzblu) [inline web/views/pages/index.ejs]: I don t know if that on purpose but you don t have anymore Electricity map | before the title

Comment (nikkozzblu) [inline web/views/pages/index.ejs]: Ok no sorry, only meta

**Pull request merged**